src/child/actions/window.js

```diff
@@ -75,6 +82,54 @@ function updatingOptionsError() {
     };
 }
 
+export function resizeToMin() {
```

Comment:
`minimise`, `maximise`, and `restore` are dispatched by [these event listeners](https://github.com/jdhodges/StockFlux/blob/fea2287d2244886634e6d03fffd9f9a43a16fcd8/src/child/containers/toolbar/Toolbar.js#L33-L35). This means they are dispatched when minimising, maximising, or restoring from either the `Toolbar` or the OS. I couldn't think of another way to cover both those cases so hopefully this is ok.